lib/zlib.js

```diff
@@ -300,7 +301,8 @@ function Zlib(opts, mode) {
   Transform.call(this, opts);
   this.bytesWritten = 0;
   this._handle = new binding.Zlib(mode);
-  this._handle.jsref = this; // Used by processCallback() and zlibOnError()
+  // Used by processCallback() and zlibOnError()
+  this._handle[owner_symbol] = this;
```

Comment:
Maybe use a non-enumerable property here? Or would this not show up when inspecting the object?

Reply:
It would, but it would show up only when inspecting the *handle* object. In that case, I think pointing to the JS wrapper is a good idea.

When inspecting the JS wrapper, this shows up as `[Circular]`, which seems okay to me.

Reply:
In that case `_handle` is probably the thing to hide, not the owner on the handle.

One thing I know I’ve thought about a while ago is a custom inspection function for all native handles – not something that blends them out completely, but rather gives a useful overview over the current state in some way.